dlpy/tests/test_model.py

```diff
@@ -908,6 +908,19 @@ def test_imagescaler2(self):
         self.assertAlmostEqual(onnx_model.graph.node[0].attribute[0].floats[2], 0.3)
         self.assertAlmostEqual(onnx_model.graph.node[0].attribute[1].f, 1/255.)
 
+    def test_load_reshape_detection(self):
+        if self.data_dir is None:
+            unittest.TestCase.skipTest(self, "DLPY_DATA_DIR is not set in the environment variables")
+        yolo_model = Model(self.s)
+        yolo_model.load(self.data_dir + 'YOLOV2_MULTISIZE.sashdat')
```

Comment:
do we have this data somewhere? how large it is? if small we can put under test/datasources